workspace/dao.py

```diff
@@ -0,0 +1,86 @@
+from db_python_util.db_classes import Workspace
+from db_python_util.db_helper import ConnectionManager
+
+@ConnectionManager.requires_connection
+def create(name, owner):
+    """
+    Create a new empty Workspace
+    """
+
+    workspace = Workspace(name = name, users = [owner], task_types = [], tasks = [])
+    workspace.save()
+
+    return workspace.pk()
```

Comment:
since we didn't get any type annotations it's hard to say but I suspect the service layer is expecting a string here >.<

maybe something like this?
```suggestion
    return workspace.id.binary.hex()
```